src/TimeOfDayFilter.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace TraderInteractive\Filter;
+
+use TraderInteractive\Exceptions\FilterException;
+
+final class TimeOfDayFilter
+{
+    /**
+     * @var string
+     */
+    const NON_EMPTY_STRING_ERROR = '$value must be a non-empty string';
+
+    /**
+     * @var string
+     */
+    const INCORRECT_FORMAT_ERROR = '$value must be in the correct format HH:MM:SS';
```

Comment:
This is just a suggestion, but I'm not a fan of using PHP syntax in filter messages, since they may be shown to the user.
```suggestion
    const INCORRECT_FORMAT_ERROR = 'Value must be in the format HH:MM:SS';
```